love/src/components/AuxTel/Dome/DomeShutter.jsx

```diff
@@ -136,16 +138,32 @@ export default class DomeShutter extends Component {
           </g>
 
           {/* Main door */}
-          <rect
-            x={x0 - extraApperture - (mainDoorWidth * this.props.mainDoorOpeningPercentage) / 100}
-            y={y0 - rSinAlpha}
-            width={mainDoorWidth}
-            height={2 * rSinAlpha}
-            fill="white"
-            fillOpacity="0.2"
-            stroke="white"
-            strokeWidth="2"
-          />
+          <g clipPath={`circle(${r}px at center)`} style={{ mask: 'url(#domeMask)' }}>
```

Comment:
We should use an unique id here. I feel this could lead to errors when having multiple views of the same type.

Reply:
I test this case and it's not have problem

Reply:
It's a mask for hide the doors out of circle of dome. For the reduce the case that its used the same name in other css, added the unique id.